EasyPost/Models/API/Address.cs

```diff
@@ -117,13 +117,6 @@ public class Address : EasyPostObject, Parameters.IAddressParameter
         public string? Zip { get; set; }
 
         #endregion
-
-        /// <summary>
-        ///     Initializes a new instance of the <see cref="Address"/> class.
-        /// </summary>
-        internal Address()
```

Comment:
question: how does this work? These inherit the public constructor of the EasyPostObject?

Reply:
A class with no explicit constructor inherently has a public constructor.